src/DropDownMenu/DropDownMenu.js

```diff
@@ -364,6 +370,7 @@ class DropDownMenu extends Component {
             {displayValue}
           </div>
           <IconButton
+            disableTouchRipple={disableTouchRipple === undefined ? disabled : disableTouchRipple}
```

Comment:
So then, what I need to is to pass `<DropDownMenu />`'s `disabled` prop to `<IconButton />`, which would then disable the ripple effect?